Tiltfile

```diff
@@ -80,7 +80,7 @@ if settings.get('debug').get('enabled'):
 
 
 docker_build_with_restart(
-    'ghcr.io/external-secrets/external-secrets',
+    'localhost:5000/external-secrets',
```

Comment:
This should stay ghcr, since that is how Tilt will find the right container to replace. :)